<launch>
    <node pkg="local_log" type="local_logger.py" name="system_logging"/>
</launch>

